internal/store/storageclass.go

```diff
@@ -35,8 +36,20 @@ var (
 			Type: metric.Gauge,
 			Help: "Information about storageclass.",
 			GenerateFunc: wrapStorageClassFunc(func(s *storagev1.StorageClass) *metric.Family {
+
+				// Add default values if missing.
+				if s.ReclaimPolicy == nil {
+					s.ReclaimPolicy = new(v1.PersistentVolumeReclaimPolicy)
```

Comment:
Let's define a variable named `defaultReclaimPolicy` which will hold this value. You can then assign `s.ReclaimPolicy` the pointer to the `defaultReclaimPolicy` variable.

And let's do the same for `VolumeBindingMode`

Reply:
We won't need to use `new` either.